include/hse/hse.h

```diff
@@ -1156,17 +1159,17 @@ hse_kvs_cursor_seek_range(
     const void **          found,
     size_t *               found_len);
 
-/** @brief Update a the cursor view.
+/** @brief Update the view of a cursor.
  *
  * This operation updates the snapshot view of a non-transaction cursor. It is a
  * no-op on transaction cursors.
  *
```

Comment:
Note that here we are correctly using the terms "non-transaction cursor" and "transaction cursor".